.github/workflows/release.yml

```diff
@@ -45,8 +43,8 @@ jobs:
         name: 'Authenticate to Google Cloud'
         uses: 'google-github-actions/auth@c4799db9111fba4461e9f9da8732e5057b394f72' # ratchet:google-github-actions/auth@v0
         with:
-          workload_identity_provider: '${{ env.WIF_PROVIDER }}'
-          service_account: '${{ env.WIF_SERVICE_ACCOUNT }}'
+          workload_identity_provider: '${{ vars.WIF_PROVIDER }}'
```

Comment:
Should we make this the production environment also?

Reply:
oh, I missed those, I'll update them. I'm eventually going to rip prod deployment out of this workflow and into infra.

Reply:
oh, I missed those, I'll update them. I'm eventually going to rip prod deployment out of this workflow and into infra.

Reply:
can you do it now so we dont break existing production?

Reply:
also, theres probably some IAM permissions in infra needed here.

Reply:
great, more PRs! :)